docs/feel-expression.md

```diff
@@ -338,6 +338,8 @@ function(x,y) external {
 }
 ```
 
+:warning: External functions are disabled by default (security risk). Use the [FunctionProvider API](function-provider-spi) instead or enable external functions in the configuration (not recommended).
```

Comment:
Unclear what "enable functions in the configuration" refers to. Maybe an example of how to do that would help

Reply:
PS: became clear after doing the whole review. But just reading the docs it left me puzzled and misdirected me into looking for a configuration file.

Reply:
Agree. The configuration (or how to use/setup the engine) is not well documented yet.